tutorials/ChannelMiddleware.afterAction.md

```diff
@@ -0,0 +1,41 @@
+Register middleware that fires after the action. If the action does not return the payload, then the afterAction will not have access to the payload object.
+
+### Channel Provider
+```js
+(async ()=> {
+    const provider = await fin.InterApplicationBus.Channel.create('channelName');
+
+    await provider.register('provider-action', (payload, identity) => {
+        return {
+            echo: payload
+        };
+    });
+
+    await provider.afterAction((action, payload, identity) => {
+        //the payload can be altered here before handling the action.
+        payload.sent = date.now();
```

Comment:
`//the payload can be altered here before handling the action.` 

becomes > 
`//the payload can be altered here after handling the action but before sending an acknowledgement.`